fenix/app/src/androidTest/java/org/mozilla/fenix/ui/BookmarksTest.kt

```diff
@@ -177,9 +118,10 @@ class BookmarksTest {
         }
     }
 
+    // TestRail link: https://testrail.stage.mozaws.net/index.php?/cases/view/2299619
     @SmokeTest
     @Test
-    fun cancelEditBookmarkTest() {
+    fun cancelingChangesInEditModeAreNotSavedTest() {
```

Comment:
No need to keep the Smoke tag here, this is a low-priority test.

Reply:
Done !